README.md

```diff
@@ -920,6 +920,7 @@ API | Description | Auth | HTTPS | CORS |
 | [OpenUV](https://www.openuv.io) | Real-time UV Index Forecast | `apiKey` | Yes | Unknown |
 | [OpenWeatherMap](https://openweathermap.org/api) | Weather | `apiKey` | Yes | Unknown |
 | [Storm Glass](https://stormglass.io/) | Global marine weather from multiple sources | `apiKey` | Yes | Yes |
+| [Troposphere.io](https://www.troposphere.io/developer) | Global weather and climate data | `apiKey` | Yes | Yes |
```

Comment:
```suggestion
| [Troposphere](https://www.troposphere.io/developer) | Global weather and climate data | `apiKey` | Yes | Yes |
```

Reply:
No need to add the TLD as it was already there in the [URL](https://www.troposphere.io/developer)